templates/archive-tsml-ui-classic.php

```diff
@@ -0,0 +1,27 @@
+ <!-- USING TSML UI IN A CLASSIC THEME -->
+<?php
+
+get_header();
+
+if (is_active_sidebar('tsml_meetings_top')) {  ?>
+    <div class="widgets meetings-widgets meetings-widgets-top" role="complementary">
+        <?php dynamic_sidebar('tsml_meetings_top') ?>
+    </div>
+<?php } ?>
+
+<div class="wp-site-blocks">
+
+<?php  
+
+echo tsml_ui();  ?>
+
+</div>
+<?php if (is_active_sidebar('tsml_meetings_bottom')) { ?>
+    <div class="widgets meetings-widgets meetings-widgets-bottom" role="complementary">
+        <?php dynamic_sidebar('tsml_meetings_bottom') ?>
+    </div>
+<?php } 
+
+get_footer();
+
+?>
```

Comment:
it's a best practice not to end your php file with `?>` -- doing so allows whitespace to sneak into the output when sometimes that's not desired